azure/components/userpool/id.ftl

```diff
@@ -0,0 +1,51 @@
+[#ftl]
+
+[#-- Azure does not have a concept of userpool's as         --]
+[#-- other providers (AWS) have. In order to perform any    --]
+[#-- action in Azure you must already have an Azure Active  --]
+[#-- Directory (AAD) account. Thus userpools themselves     --]
+[#-- do not perform any configuration, however the child    --]
+[#-- components - Auth Provider and Client - do have Azure  --]
+[#-- equivalents and so do generate content.                --]
+
+[@addResourceGroupInformation
+    type=USERPOOL_COMPONENT_TYPE
+    attributes=[]
+    provider=AZURE_PROVIDER
+    resourceGroup=DEFAULT_RESOURCE_GROUP
+    services=[
+        AZURE_AAD_USERPOOL_PSEUDO_SERVICE
+    ]
+/]
+
+[@addResourceGroupInformation
+    type=USERPOOL_AUTHPROVIDER_COMPONENT_TYPE
+    attributes=[]
+    provider=AZURE_PROVIDER
+    resourceGroup=DEFAULT_RESOURCE_GROUP
+    services=[
+        AZURE_AAD_USERPOOL_PSEUDO_SERVICE
+    ]
+/]
+
+[@addResourceGroupInformation
+    type=USERPOOL_CLIENT_COMPONENT_TYPE
+    attributes=[
+        {
+            "Names" : "AllowOtherTenants",
+            "Description" : "Application can be used by other Azure AD Tenants.",
+            "Type" : BOOLEAN_TYPE,
+            "Default" : false
+        },
+        {
+            "Names" : "ReplyUrls",
+            "Description" : "URI's to which Azure AD will redirect in response to an OAuth 2.0 Request",
+            "Type": ARRAY_OF_STRING_TYPE
```

Comment:
We actually have these implmented based on links to other components in AWS rather than configuration of the pool itself. 

So the user pool links across to things it authenticates to get their url details.

Reply:
addressed this in 8586e25